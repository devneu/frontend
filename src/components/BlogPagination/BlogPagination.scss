.blog__pagination{
   text-align: center;
}