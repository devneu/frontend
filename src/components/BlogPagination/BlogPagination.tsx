import React from 'react';
import { Pagination } from 'antd';
import './BlogPagination.scss'


const BlogPagination = () => {
   return (
      <Pagination defaultCurrent={1} total={50} />
   )
}

export default BlogPagination;