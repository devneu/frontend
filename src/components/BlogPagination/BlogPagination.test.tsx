import BlogPagination from './BlogPagination';
import renderer from 'react-test-renderer';

describe('BlogPagination', () => {
   it('Should successfully render', () => {
      const wrapper = renderer
         .create(
            <BlogPagination />
         )
         .toJSON();
      expect(wrapper).toMatchSnapshot();
   });
});