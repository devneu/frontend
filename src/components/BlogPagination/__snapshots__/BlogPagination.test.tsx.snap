// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlogPagination Should successfully render 1`] = `
<ul
  className="ant-pagination blog__pagination"
  style={Object {}}
  unselectable="unselectable"
>
  <li
    aria-disabled={true}
    className="ant-pagination-prev ant-pagination-disabled"
    onClick={[Function]}
    onKeyPress={[Function]}
    tabIndex={null}
    title="Previous Page"
  >
    <a
      disabled={true}
    >
      Newer
    </a>
  </li>
  <li
    className="ant-pagination-item ant-pagination-item-1 ant-pagination-item-active"
    onClick={[Function]}
    onKeyPress={[Function]}
    tabIndex="0"
    title={1}
  >
    <a
      rel="nofollow"
    >
      1
    </a>
  </li>
  <li
    className="ant-pagination-item ant-pagination-item-2"
    onClick={[Function]}
    onKeyPress={[Function]}
    tabIndex="0"
    title={2}
  >
    <a
      rel="nofollow"
    >
      2
    </a>
  </li>
  <li
    className="ant-pagination-item ant-pagination-item-3"
    onClick={[Function]}
    onKeyPress={[Function]}
    tabIndex="0"
    title={3}
  >
    <a
      rel="nofollow"
    >
      3
    </a>
  </li>
  <li
    className="ant-pagination-item ant-pagination-item-4"
    onClick={[Function]}
    onKeyPress={[Function]}
    tabIndex="0"
    title={4}
  >
    <a
      rel="nofollow"
    >
      4
    </a>
  </li>
  <li
    className="ant-pagination-item ant-pagination-item-5"
    onClick={[Function]}
    onKeyPress={[Function]}
    tabIndex="0"
    title={5}
  >
    <a
      rel="nofollow"
    >
      5
    </a>
  </li>
  <li
    className="ant-pagination-item ant-pagination-item-6"
    onClick={[Function]}
    onKeyPress={[Function]}
    tabIndex="0"
    title={6}
  >
    <a
      rel="nofollow"
    >
      6
    </a>
  </li>
  <li
    aria-disabled={false}
    className="ant-pagination-next"
    onClick={[Function]}
    onKeyPress={[Function]}
    tabIndex={0}
    title="Next Page"
  >
    <a
      disabled={false}
    >
      Older
    </a>
  </li>
  <li
    className="ant-pagination-options"
  >
    <div
      className="ant-select ant-pagination-options-size-changer ant-select-single ant-select-show-arrow"
      onBlur={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
    >
      <div
        className="ant-select-selector"
        onClick={[Function]}
        onMouseDown={[Function]}
      >
        <span
          className="ant-select-selection-search"
        >
          <input
            aria-activedescendant="undefined_list_0"
            aria-autocomplete="list"
            aria-controls="undefined_list"
            aria-haspopup="listbox"
            aria-owns="undefined_list"
            autoComplete="off"
            className="ant-select-selection-search-input"
            onChange={[Function]}
            onCompositionEnd={[Function]}
            onCompositionStart={[Function]}
            onKeyDown={[Function]}
            onMouseDown={[Function]}
            onPaste={[Function]}
            readOnly={true}
            role="combobox"
            style={
              Object {
                "opacity": 0,
              }
            }
            type="search"
            unselectable="on"
            value=""
          />
        </span>
        <span
          className="ant-select-selection-item"
          title="10 / page"
        >
          10 / page
        </span>
      </div>
      <span
        aria-hidden={true}
        className="ant-select-arrow"
        onMouseDown={[Function]}
        style={
          Object {
            "WebkitUserSelect": "none",
            "userSelect": "none",
          }
        }
        unselectable="on"
      >
        <span
          aria-label="down"
          className="anticon anticon-down ant-select-suffix"
          role="img"
        >
          <svg
            aria-hidden="true"
            data-icon="down"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M884 256h-75c-5.1 0-9.9 2.5-12.9 6.6L512 654.2 227.9 262.6c-3-4.1-7.8-6.6-12.9-6.6h-75c-6.5 0-10.3 7.4-6.5 12.7l352.6 486.1c12.8 17.6 39 17.6 51.7 0l352.6-486.1c3.9-5.3.1-12.7-6.4-12.7z"
            />
          </svg>
        </span>
      </span>
    </div>
  </li>
</ul>
`;
